kpops/component_handlers/kubernetes/utils.py

```diff
@@ -0,0 +1,28 @@
+import hashlib
+import logging
+
+log = logging.getLogger("K8sUtils")
+
+
+def trim(max_len: int, name: str, suffix: str) -> str:
```

Comment:
If this is under `kubernetes` folder would it make sense to move it under `KubernetesApp` class?

Reply:
I don't have a preference, I just kept it the same way as the previous Helm trimming function which was part of Helm wrapper utils.

Reply:
Okay, we can perspectively refactor it there. I think it is cleaner.